TSPSolver.py

```diff
@@ -280,9 +280,20 @@ def initializeNextState(self, prevState, cityID, city):
 	def __lt__(self, other):
 		return self.path[-1] < other.path[-1]
 
+	def mutate(self, genome):
```

Comment:
Shouldn't this and the fitness function be part of the Genome class?

Reply:
I don't think it really matters. Either way the data will be getting changed.

Reply:
Ya, they could just be static. I was wondering about them being included as part of the State class instead of the TSPSolver or Genome

Reply:
It works either way for me, just as long as it works.